### PR TITLE
fix: fix cluster test

### PR DESCRIPTION
**Pull request by dranikpg**
The test didn't check anything because we pushed the config before actually generating the capture. Note that in my original #2496 this is not the case 🤨

### CODE REVIEW

File: tests/dragonfly/cluster_test.py [hunk 1]
```diff
@@ -1104,12 +1116,12 @@ async def list_counter(key, client: aioredis.RedisCluster):
     for counter in counters:
         counter.cancel()
 
-    # Push new config
-    await push_config(json.dumps(await generate_config()), [node.admin_client for node in nodes])
-
     # Generate capture, capture ignores counter keys
     capture = await seeder.capture()
```
Comment (adiholden) [inline tests/dragonfly/cluster_test.py:1120]: Acually I think we should call capture before we issue SLOT-MIGRATION-FINALIZE

Comment (dranikpg) [reply tests/dragonfly/cluster_test.py:1120]: We can capture before and after, consistency shouldn't break for outside observers. We could even capture 3 times and check it's the same every time

**Review by chakaz — APPROVED** [review]
LG. unfortunately, I've been working on the same area doing pretty much the same thing :|

**Comment by dranikpg** [comment]
I was doing some screenshots for a testing presentation and wanted to show this test as an example. But then I noticed it was wrong 😂

**Review by adiholden — APPROVED** [review]

**Pull request merged**